client/src/components/AssignmentList.tsx

```diff
@@ -57,7 +57,7 @@ interface AssignmentListProps {
   list: Assignment[]
   sortMethod?: string
   filterCriteria?: string
-  onDelete: (assignmentId: string) => Promise<any>
+  onDelete: (assignmentId: string) => Promise<void>
```

Comment:
this forbids to return something. if we just not need something here, unknown would be better, right?

Reply:
Yeah good point. It did this everywhere else too.